src/client/entity-editor/entity-editor.tsx

```diff
@@ -27,13 +27,15 @@ import NameSection from './name-section/name-section';
 import RelationshipSection from './relationship-editor/relationship-section';
 import SubmissionSection from './submission-section/submission-section';
 import _ from 'lodash';
+import {getEntityUrl} from '../helpers/entity';
 import {submit} from './submission-section/actions';
 
 
 type OwnProps = {
 	children: React.ReactElement<any>,
 	heading: string,
 	intitialState:Record<string, any>,
+	entity: any
```

Comment:
I don't know where we get this prop from, but it seems to be working ❓

Reply:
so the idea of using this prop came to me when I looked at author.js which use getEntityUrl( ) to etract the Url prefix for a edit button which redirect us to edit page 

https://github.com/metabrainz/bookbrainz-site/blob/master/src/client/components/pages/entities/author.js#L197

honestly , when I started working on this it was quite difficult to understand how data is being passed to pages in our website when they loads , but after some tracing I figure out that the most probably the entity prop is most probably coming from here
https://github.com/metabrainz/bookbrainz-site/blob/master/src/client/helpers/props.js#L56

Reply:
sorry for my confusion as i am still understanding the BB codebase .... so later i found out that entity editor component is getting its props from `src/client/entity-editor/controller.js` file from this part [https://github.com/metabrainz/bookbrainz-site/blob/master/src/client/entity-editor/controller.js/#L49-L51](https://github.com/metabrainz/bookbrainz-site/blob/master/src/client/entity-editor/controller.js/#L49-L51)

I hope this time I am right but pls do correct if I am wrong here 😅